go.mod

```diff
@@ -1,43 +1,151 @@
 module github.com/axelarnetwork/axelar-core
 
-go 1.16
+go 1.17
 
 require (
-	github.com/armon/go-metrics v0.3.9
-	github.com/axelarnetwork/tm-events v0.0.0-20211102054919-baac5ca41de2
-	github.com/axelarnetwork/utils v0.0.0-20211102051812-9d1028976360
+	github.com/armon/go-metrics v0.3.10
+	github.com/axelarnetwork/tm-events v0.0.0-20220103213714-77c3c8df16d3
+	github.com/axelarnetwork/utils v0.0.0-20211119234529-fb861e5fd25f
 	github.com/btcsuite/btcd v0.22.0-beta
 	github.com/btcsuite/btcutil v1.0.3-0.20201208143702-a53e38424cce
-	github.com/cosmos/cosmos-sdk v0.44.3
-	github.com/cosmos/ibc-go v1.2.0
-	github.com/ethereum/go-ethereum v1.10.11
+	github.com/cosmos/cosmos-sdk v0.44.5
+	github.com/cosmos/ibc-go v1.2.5
+	github.com/ethereum/go-ethereum v1.10.14
 	github.com/gogo/protobuf v1.3.3
 	github.com/golang/protobuf v1.5.2
 	github.com/gorilla/mux v1.8.0
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
-	github.com/klauspost/compress v1.11.9 // indirect
 	github.com/matryer/moq v0.2.5
 	github.com/miguelmota/go-ethereum-hdwallet v0.1.1
 	github.com/rakyll/statik v0.1.7
-	github.com/rcrowley/go-metrics v0.0.0-20201227073835-cf1acfcdf475 // indirect
 	github.com/regen-network/cosmos-proto v0.3.1
-	github.com/rs/zerolog v1.23.0
-	github.com/spf13/cast v1.3.1
-	github.com/spf13/cobra v1.2.1
+	github.com/rs/zerolog v1.26.1
+	github.com/spf13/cast v1.4.1
+	github.com/spf13/cobra v1.3.0
 	github.com/spf13/pflag v1.0.5
-	github.com/spf13/viper v1.8.1
+	github.com/spf13/viper v1.10.1
 	github.com/stretchr/testify v1.7.0
-	github.com/tendermint/tendermint v0.34.14
-	github.com/tendermint/tm-db v0.6.4
-	github.com/tyler-smith/go-bip39 v1.1.0 // indirect
-	golang.org/x/crypto v0.0.0-20210921155107-089bfa567519 // indirect
-	golang.org/x/sys v0.0.0-20211101204403-39c9dd37992c // indirect
-	golang.org/x/text v0.3.6 // indirect
-	google.golang.org/genproto v0.0.0-20210602131652-f16073e35f0c
-	google.golang.org/grpc v1.40.0
+	github.com/tendermint/tendermint v0.34.15
+	github.com/tendermint/tm-db v0.6.6
+	golang.org/x/text v0.3.7
+	google.golang.org/genproto v0.0.0-20211223182754-3ac035c7e7cb
+	google.golang.org/grpc v1.43.0
 	google.golang.org/protobuf v1.27.1
 )
 
+require (
```

Comment:
Any idea why all these indirect dependencies?

Reply:
I think it is because of this: https://stackoverflow.com/questions/69432877/why-there-are-two-require-blocks-in-go-mod-since-go-1-17